src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -171,10 +171,10 @@ public static void openPipelinePreviewMenuAndClickOnRunButton() {
   }
 
   /**
-   * Click on the Preview menu button to Open/Close the Preview menu
+   * Click on the Preview menu button to Close the Preview menu
```

Comment:
On a closer look, the `Preview` Widget can be used for both Open/Close operations. I understand, currently we might be using it for close but still better to keep it naming generic so that can be reused if we need to perform Open operation as well.

Reply:
We have separate action specific to open preview menu. 
As locator is different when preview button is active vs inactive.
https://github.com/cdapio/cdap-e2e-tests/blob/43b0d2424fb413642c88e4ada74b74bbb445f2a4/src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java#L154

Reply:
I see, makes sense.